pkg/environment/environment.go

```diff
@@ -147,6 +156,10 @@ func getEnvironmentFromParams(params kernel.Params) (Environment, error) {
 		env.BcdbPassword = e
 	}
 
+	if e := os.Getenv("ZOS_FLIST_URL"); e != "" {
+		env.FlistURL = e
```

Comment:
That's nice ! But if adding an environment variable for flist, could add one for BinRepo too ? :)

Reply:
Actually, how is this intended to be used ? There is no way to set a variable in a zinit service.

Reply:
We have others environment variable aswell defined. In the past, we talked about a way to define environment variable easily for the whole system, that was never implemented but still could be. It's easier IMO than editing service file to add custom flags (like debug)

Reply:
alright, @maxux let's open a follow up issue  for this then.